R/param_svm.R

```diff
@@ -23,6 +23,25 @@ cost <- function(range = c(-10, -1), trans = log2_trans()) {
 #' @rdname cost
 #' @export
 margin <- function(range = c(0, .2), trans = NULL) {
+  warning(
+    "The margin() function will be deprecated in versions >= 0.0.6.9000. Please
+    use svm_margin() instead. See https://github.com/tidymodels/dials/issues/85
+    for more details."
+  )
+
+  new_quant_param(
```

Comment:
To be consistent, you can just call `svm_margin(range = range, trans = trans)`

Reply:
@DavisVaughan do you mean only calling `svm_margins` instead of having both functions (margins / svm_margins) within the file?

Reply:
Our update:
```
margin <- function(range = c(0, .2), trans = NULL) {
  warning(
    "The margin() function will be deprecated in versions >= 0.0.6.9000. Please
    use svm_margin() instead. See https://github.com/tidymodels/dials/issues/85
    for more details."
  )
  
  svm_margin(range = range, trans = trans)
}
```

Reply:
^ this looks great